### PR TITLE
U2e wb s yf create delete card method

### PR DESCRIPTION
**Pull request by amaclay**

### CODE REVIEW

File: scrum/tools/sprint_math.py [hunk 1]
```diff
@@ -48,6 +48,46 @@ def request_call(url, have_headers):
     return response.json()
 
 
+def delete_card(id):
+    """Makes a DELETE request for the specified card using the requests library.
+
+    Args:
+        id: str, card id (not shortLink)
```
Comment (MPBeholder) [inline scrum/tools/sprint_math.py]: Can you add an example? i.e `gGnafhow` vs `gGnafhow/2557-internal-work-removal-of-dead-link-in-specialty-admission` etc?

File: scrum/tools/sprint_math.py [hunk 2]
```diff
@@ -48,6 +48,46 @@ def request_call(url, have_headers):
...
+
+    Usage:
+        # to delete cards older than <date>
+        from datetime import datetime
```
Comment (MPBeholder) [inline scrum/tools/sprint_math.py:59]: What's with the python in comments? Are these suggestions on how to tweak this to target other boards?

**Review by MPBeholder — CHANGES_REQUESTED** [review]

File: scrum/tools/sprint_math.py (hunk 2 above)
Comment (amaclay) [reply scrum/tools/sprint_math.py:59]: It's the loop I ran that called this method

File: scrum/tools/sprint_math.py (hunk 1 above)
Comment (amaclay) [reply scrum/tools/sprint_math.py]: updated

**Review by MPBeholder — APPROVED** [review]

**Pull request merged**